src/swell/utilities/slurm.py

```diff
@@ -184,6 +184,11 @@ def prepare_scheduling_dict(
             validate_directives(model_directives)
             scheduling_dict[slurm_task]["directives"][model_component] = model_directives
 
+        x = None
+        if slurm_task in experiment_task_directives.keys():
+            x = experiment_task_directives[slurm_task].get('execution_time_limit', None)
+        scheduling_dict[slurm_task]['execution_time_limit'] = x
+
```

Comment:
This logic seems overly convoluted. I suggest just doing this and eliminating the change in `create_experiment.py`.

```suggestion
        if slurm_task in experiment_task_directives.keys():
            scheduling_dict[slurm_task]["execution_time_limit"] = (
                experiment_task_directives[slurm_task].get("execution_time_limit", "PT1H")
            )
```

The `dict.get(key, default_value)` method will grab `dict[key]` if `key` is in `dict` or, if `key` is not in `dict`, will return `default_value` instead.

Reply:
Good suggestion, and the modification in `create_experiment.py` is consolidated to here!